docs/docs/integrations/retrievers/bm25.ipynb

```diff
@@ -132,13 +153,68 @@
     "result"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "51043723814c0d68",
+   "metadata": {},
+   "source": [
+    "## Preprocessing Function\n",
+    "Pass a custom preprocessing function to the retriever to improve search results. Tokenizing text at the word level can enhance retrieval, especially when using vector stores like Chroma, Pinecone, or Faiss for chunked documents."
```

Comment:
Might make sense to explain more what the preprocessor is tokenizing (e.g., document content, query, both...)